Doc/library/enum.rst

```diff
@@ -761,11 +761,11 @@ Data Types
 Supported ``__dunder__`` names
 """"""""""""""""""""""""""""""
 
-:attr:`__members__` is a read-only ordered mapping of ``member_name``:``member``
+:attr:`!EnumType.__members__` is a read-only ordered mapping of ``member_name``:``member``
```

Comment:
I don't really get this either. This is a list of supported dunder and sunder names, and none of the other names have the class name explicitly prefixed, so I'm not sure why it makes sense to treat this differently.

Either `__members__` should be explicitly documented under `EnumType`, like a number of other dunders and sunders are (including some listed here), and the link fixed here in the meantime (which displays the same as now until it is added):

```suggestion
:attr:`~EnumType.__members__` is a read-only ordered mapping of ``member_name``:``member``
```

or, if for whatever reason it doesn't make sense to explicitly document `__members__` in the usual way, silence the warning without the prefix:

```suggestion
:attr:`!__members__` is a read-only ordered mapping of ``member_name``:``member``
```

Reply:
Let's document `__member__` separately and keep the focus of this PR *focused* 😉

Reply:
Sure, but if `__members__` should be documented (or the decision left until later), then deresolving it not only silences a _valid_ warning, but also means that it will stay broken even if and when `__members__` is documented, rather than start working automatically.

Therefore, unless @ethanfurman has some reason to not explicitly document `__members__` but still list it here (which is possible), then the default approach should be the former, i.e. to fix it but leave it resolving, unless and until it is decided to explicitly leave it undocumented.